test/CMakeLists.txt

```diff
@@ -69,4 +69,9 @@ if(MANIFOLD_FUZZ)
   target_link_libraries(polygon_fuzz PUBLIC manifold)
   link_fuzztest(polygon_fuzz)
   gtest_discover_tests(polygon_fuzz)
+
+  add_executable(manifold_fuzz manifold_fuzz.cpp)
+  target_link_libraries(manifold_fuzz PUBLIC manifold)
+  link_fuzztest(manifold_fuzz)
+  gtest_discover_tests(manifold_fuzz)
```

Comment:
Are we going to enable `MANIFOLD_FUZZ` on the CI?

Reply:
No, this is something that runs indefinitely, and non-deterministically. So this is probably something we want to run in our own machines.